### PR TITLE
Attributes description

### PR DESCRIPTION
**Pull request by zdne**
Update protagonist to include attributes description as implemented in Snow Crash

### CODE REVIEW

**Comment by zdne** [comment]
@pksunkara I have turned it into PR, please let me know when you are done with the review and I will address the comments. Thanks!

**Comment by pksunkara** [comment]
> Can't be done because that serializer is way too coupled with stream buffer serialization and all the base methods are using ostream

True. We need to change that later.

File: tools/protagonist.js [hunk 1]
```diff
@@ -8,8 +8,8 @@ var protagonist = require('../build/Release/protagonist');
 //var protagonist = require('protagonist');
 
 // Add Segfault handler
-var segfaultHandler = require('segfault-handler');
-segfaultHandler.registerHandler();
+// var segfaultHandler = require('segfault-handler');
+// segfaultHandler.registerHandler();
```
Comment (pksunkara) [inline tools/protagonist.js]: I don't understand why these are removed. If you want to remove them completely, please remove them from package.json too.

**Comment by pksunkara** [comment]
Reviewed.

**Comment by zdne** [comment]
@pksunkara comments addressed

**Pull request merged**